Neos.Utility.Arrays/Classes/Arrays.php

```diff
@@ -194,7 +194,31 @@ public static function array_reduce(array $array, string $function, $initial = n
     public static function getValueByPath(array &$array, $path)
     {
         if (is_string($path)) {
-            $path = explode('.', $path);
+            if (strpos($path, '\\.') === -1) {
```

Comment:
How can this return -1? According to https://www.php.net/manual/en/function.strpos.php this "Returns FALSE if the needle was not found."

Reply:
`strpos ( string $haystack , mixed $needle [, int $offset = 0 ] ) : **int**` huh? PHP 🤷‍♂ But you are right.
```suggestion
            if (strpos($path, '\\.') === false) {
```

Reply:
Yeah.. One of the reasons for https://wiki.php.net/rfc/union_types_v2#false_pseudo-type